src/bisect.cpp

```diff
@@ -132,23 +153,104 @@ std::vector<long> bisect_multilevel(bulk::world& world,
         }
 
 
-        std::map<std::vector<long>, int> edge_counts;
-        std::map<std::vector<long>, long> ids;
-        std::unordered_set<long> remove_nets;
+        if (simplify_mode == "complete") {

```

Comment:
Did this improve the running time?

Reply:
Simplification is not beneficial if the hypergraph does not yet have duplicates. So when the hypergraph is too small to be reduced sufficiently by parallel coarsening it makes no sense to do simplification.